src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/wrapper/PKCS11Exception.java

```diff
@@ -63,229 +63,175 @@
+        CKR_OPERATION_ACTIVE(0x00000090L),
+        CKR_OPERATION_NOT_INITIALIZED(0x00000091L),
+        CKR_PIN_INCORRECT(0x000000A0L),
+        CKR_PIN_INVALID(0x000000A1L),
+        CKR_PIN_LEN_RANGE(0x000000A2L),
+        CKR_PIN_EXPIRED(0x000000A3L),
+        CKR_PIN_LOCKED(0x000000A4L),
+        CKR_SESSION_CLOSED(0x000000B0L),
+        CKR_SESSION_COUNT(0x000000B1L),
+        CKR_SESSION_HANDLE_INVALID(0x000000B3L),
+        CKR_SESSION_PARALLEL_NOT_SUPPORTED(0x000000B4L),
+        CKR_SESSION_READ_ONLY(0x000000B5L),
+        CKR_SESSION_EXISTS(0x000000B6L),
+        CKR_SESSION_READ_ONLY_EXISTS(0x000000B7L),
+        CKR_SESSION_READ_WRITE_SO_EXISTS(0x000000B8L),
+        CKR_SIGNATURE_INVALID(0x000000C0L),
+        CKR_SIGNATURE_LEN_RANGE(0x000000C1L),
+        CKR_TEMPLATE_INCOMPLETE(0x000000D0L),
+        CKR_TEMPLATE_INCONSISTENT(0x000000D1L),
+        CKR_TOKEN_NOT_PRESENT(0x000000E0L),
+        CKR_TOKEN_NOT_RECOGNIZED(0x000000E1L),
+        CKR_TOKEN_WRITE_PROTECTED(0x000000E2L),
+        CKR_UNWRAPPING_KEY_HANDLE_INVALID(0x000000F0L),
+        CKR_UNWRAPPING_KEY_SIZE_RANGE(0x000000F1L),
+        CKR_UNWRAPPING_KEY_TYPE_INCONSISTENT(0x000000F2L),
+        CKR_USER_ALREADY_LOGGED_IN(0x00000100L),
+        CKR_USER_NOT_LOGGED_IN(0x00000101L),
+        CKR_USER_PIN_NOT_INITIALIZED(0x00000102L),
+        CKR_USER_TYPE_INVALID(0x00000103L),
+        CKR_USER_ANOTHER_ALREADY_LOGGED_IN(0x00000104L),
+        CKR_USER_TOO_MANY_TYPES(0x00000105L),
+        CKR_WRAPPED_KEY_INVALID(0x00000110L),
+        CKR_WRAPPED_KEY_LEN_RANGE(0x00000112L),
+        CKR_WRAPPING_KEY_HANDLE_INVALID(0x00000113L),
+        CKR_WRAPPING_KEY_SIZE_RANGE(0x00000114L),
+        CKR_WRAPPING_KEY_TYPE_INCONSISTENT(0x00000115L),
+        CKR_RANDOM_SEED_NOT_SUPPORTED(0x00000120L),
+        CKR_RANDOM_NO_RNG(0x00000121L),
+        CKR_DOMAIN_PARAMS_INVALID(0x00000130L),
+        CKR_CURVE_NOT_SUPPORTED(0x00000140L),
+        CKR_BUFFER_TOO_SMALL(0x00000150L),
+        CKR_SAVED_STATE_INVALID(0x00000160L),
+        CKR_INFORMATION_SENSITIVE(0x00000170L),
+        CKR_STATE_UNSAVEABLE(0x00000180L),
+        CKR_CRYPTOKI_NOT_INITIALIZED(0x00000190L),
+        CKR_CRYPTOKI_ALREADY_INITIALIZED(0x00000191L),
+        CKR_MUTEX_BAD(0x000001A0L),
+        CKR_MUTEX_NOT_LOCKED(0x000001A1L),
+        CKR_NEW_PIN_MODE(0x000001B0L),
+        CKR_NEXT_OTP(0x000001B1L),
+        CKR_EXCEEDED_MAX_ITERATIONS(0x000001B5L),
+        CKR_FIPS_SELF_TEST_FAILED(0x000001B6L),
+        CKR_LIBRARY_LOAD_FAILED(0x000001B7L),
+        CKR_PIN_TOO_WEAK(0x000001B8L),
+        CKR_PUBLIC_KEY_INVALID(0x000001B9L),
+        CKR_FUNCTION_REJECTED(0x00000200L),
+        CKR_TOKEN_RESOURCE_EXCEEDED(0x00000201L),
+        CKR_OPERATION_CANCEL_FAILED(0x00000202L),
+        CKR_VENDOR_DEFINED(0x80000000L);
+
+        private final long value;
+
+        RV(long value) {
+            this.value = value;
         }
-    }
+        long value() {
+            return value;
+        }
```

Comment:
I guess it is safe to use the 'value' variable directly for enum.  The value() method may be not necessary.  Just a personal preference, ignore it if you like the current way.

Reply:
Sure, I like it too. Will update.

Reply:
Thank you.  I have no more comment.